### ISSUE TITLE
Invalid font-family for editor

### ISSUE THREAD

**Issue by BramVer** [BoostIO/Boostnote#1358]
I am experiencing an issue where the font for the editor is by default always set to an invalid value, instead of the font I specify in the settings. This is not the case for the preview font-family, so it's not as if the font could not be found or is provided incorrectly.
The value provided is always the following:
```css
font-family: MarkdownEditor__codeEditor___browser-components-, Monaco, Menlo, "Ubuntu Mono", Consolas, source-code-pro, monospace;
```

This issue was introduced in the previous version (0.8.18).
Even after upgrading to the newest version which supposedly fixes being able to set fonts from settings.

**OS**: Arch Linux
**Version**: 0.8.19-1
**Package**: https://aur.archlinux.org/packages/boostnote/

**Settings**:
![image](https://user-images.githubusercontent.com/16743200/34478077-1fbfb782-ef9f-11e7-955f-fc5bd0c09e1f.png)

**Issue**:
![image](https://user-images.githubusercontent.com/16743200/34478093-48fc8f26-ef9f-11e7-9319-72a556caeb3e.png)

Any idea on what's going on?
So far I've been able to deal with it by opening the developer tools and changing things there, but I shouldn't have to do this each time.

**Comment by marcusstenbeck** [comment]
I'm experincing this issue too, macOS 10.10.5 and Boostnote 0.8.19

**Comment by kazup01** [comment]
We were released v0.8.20.
[Release Note](https://medium.com/boostnote/boostnote-v0-8-20-release-4e1c1b1e1dbd)
Thanks for your report @BramVer :)
